src/services/FeedSymbolChecker.ts

```diff
@@ -0,0 +1,20 @@
+import {injectable, inject} from 'inversify';
+import {Logger} from 'winston';
+
+@injectable()
+class FeedSymbolChecker {
+  @inject('Logger') private logger!: Logger;
+
+  apply(feedSymbol: string | undefined): string[] {
```

Comment:
```suggestion
  apply(feedSymbol: string | undefined): [base: string, quote: string] | undefined {
```

maybe this will be better return type?

Reply:
then we can do simply `if (!result) continue/return`

Reply:
yes, it looks more clear in this way. Thank you